### DISCUSSION TITLE
Error loading: Video returned by YouTube isn't what was requested

### DISCUSSION THREAD

**Discussion by Uru222** [jagrosh/MusicBot#1532] [Troubleshooting]
It was working fine, but out of nowhere it stopped playing YouTube songs

**Comment by MrKakan** [comment]
> It was working fine, but out of nowhere it stopped playing YouTube songs

I have had the same problem. 1/10 youtube songs can be played. Went back to 0.3.9 and youtube worked again.

**Reply by joshhmann** [reply]
Having the same issue with 5 bots. When reverting back to 0.3.9 three out of the five were working again

**Reply by nejked** [reply]
Hi I'm having the same problem, how am I meant to revert to the 0.3.9 version? Does that mean to just uninstall the latest version and installing the older version? (noob pc user)

**Reply by joshhmann** [reply]
> Hi I'm having the same problem, how am I meant to revert to the 0.3.9 version? Does that mean to just uninstall the latest version and installing the older version? (noob pc user)

You would just need to need to replace the current jar with 0.3.9

**Reply by nejked** [reply]
Oh yeah it worked, thanks!

**Reply by CripperFunny666** [reply]
3.9 just doesn't work for me. I send him a link, he enters the voice channel and immediately exits..

**Comment by jagrosh** [comment]
Duplicate of #1527
